lalrpop-util/src/lib.rs

```diff
@@ -191,6 +191,10 @@ macro_rules! lalrpop_mod {
 
     ($(#[$attr:meta])* $vis:vis $modname:ident, $source:expr) => {
         #[rustfmt::skip]
+        #[allow(clippy::extra_unused_lifetimes)]
+        #[allow(clippy::needless_lifetimes)]
```

Comment:
I understand the below 2 allows are justified as generated code and easy to avoid for human.
But is this lifetimes too?

Reply:
I think it's reasonable for the scope of this PR. As you say, from a quick glance, it might be doable, but it doesn't look totally trivial at least. If you're happy with this answer @youknowone I think you can press the green button :slightly_smiling_face: